exe/todo

```diff
@@ -29,14 +29,19 @@ class TodoCLI < Thor
     puts "\n#{Todo::Manager.new(config).display}\n"
   end #display
 
-  desc("remove [ITEM]", "removes item from list")
+  desc("remove", "removes item from list based on list number")
   option :database, default: "~/.todo"
-  def remove(item)
+  def remove
     config = {
       database: File.open(File.expand_path(options[:database]), "a+")
     }
-    #Todo::Manager.new(config).remove(item)
-    puts "Coming soon..."
+    manager = Todo::Manager.new(config)
+    puts "\n\n#{manager.display.display}"
```

Comment:
Methinks that's one too many display calls.
```suggestion
    puts "\n\n#{manager.display}"
```

Reply:
NEVER! ...ok